lib/screens/speaker/speaker_repository.dart

```diff
@@ -62,17 +62,17 @@ class APISpeakerRepository extends SpeakerRepository {
     final String tag = classTag + "getAllSpeakers()";
     http.Response response;
     try {
-      response = await sl.get<http.Client>().get(S.getSpeakerUrl);
+      response = await sl.get<http.Client>().get(S.getSpeakerUrl, headers: <String, String>{
+        "Authorization": "b3c098bcf06f5b7595bffe542caea7ff41ee3587",
```

Comment:
I don't think there is need for Authorization token to be sent. Has anyone told you to do this?

Reply:
Almost forgot after #84